### PR TITLE
Remove v1 variables code

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Continuation of https://github.com/PrefectHQ/nebula-ui/pull/4891

Removes all v1 code which was duplicated and updated for v2 to feature flag the json variables work. 

Removes all v2 keywords. Back to just "variable" everywhere.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
     "declaration": true,
-    "emitDeclarationOnly": true,
+    // "emitDeclarationOnly": true,
```
Comment (collincchoy) [inline tsconfig.json:6]: intentional? or from running tsc or smth lol

Comment (pleek91) [reply tsconfig.json:6]: oh shoot, yep didn't mean to commit that. TY

**Review by collincchoy — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (pleek91) [reply tsconfig.json]: fixed in https://github.com/PrefectHQ/prefect-ui-library/pull/2607/commits/5d0f56cea9c1886a327784458eb20e49a0e13855

**Pull request merged**